collectors/pubmed/parser.py

```diff
@@ -95,10 +96,12 @@ def parse_record(res):
     if article.get('VernacularTitle'):
         data['article_vernacular_title'] = article['VernacularTitle']['#text']
     if 'ArticleDate' in article:
-        data['article_date'] = '{year}-{month}-{day}'.format(
+        article_date = '{year}-{month}-{day}'.format(
             year=article['ArticleDate']['Year']['#text'],
             month=article['ArticleDate']['Month']['#text'],
             day=article['ArticleDate']['Day']['#text'])
+        article_date = date_parse(article_date)
```

Comment:
This is here because sometimes the `article_date` has the form `2013-Jan-21` and it was breaking. Ex: see the test document.